### PR TITLE
Add 2024 first cfp blog

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**

### CODE REVIEW

File: content/blog/2024/how-to-write-stellar-proposals.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+5. Video showcasing presentation style and link to proposal: <br />
+[Securing your python applications and network using Zero Trust Security](https://in.pycon.org/cfp/pycon-india-2023/proposals/securing-your-python-applications-and-network-using-zero-trust-security~eg1kb/) <br />
+[Accelerating testing cycles of embedded devices using Python](https://in.pycon.org/cfp/pycon-india-2023/proposals/accelerating-testing-cycles-of-embedded-devices-using-python~bqxRp/)
+
+### Other links:
+
+- [Amaze your friends by painting images on Google Sheets using Python](https://in.pycon.org/cfp/pycon-india-2023/proposals/amaze-your-friends-by-painting-images-on-google-sheets-using-python~eER80/) - expertise
+- [OpenAI Whisper and it’s amazing power to do fine-tuning demonstrated on my mother tongue](https://in.pycon.org/cfp/pycon-india-2023/proposals/openai-whisper-and-its-amazing-power-to-do-fine-tuning-demonstrated-on-my-mother-tongue~eENWa/) - unique, tailored for diverse audience 
+
```
Comment (WatchfulHitman) [inline content/blog/2024/how-to-write-stellar-proposals.md:87]: If you need any help with submitting your proposal. Please register for the upcoming mentorship call here: https://bit.ly/mnts

**Comment by Sc0rpi0n101** [comment]
Updated

File: content/blog/2024/how-to-write-stellar-proposals.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+### Here are some tips to enhance the proposal and make it stand out:
+
+- Make your proposal unique by offering fresh insights, innovative solutions, and original perspectives. Avoid repeating previous ideas and plagiarism. 
+- Tailor your content to diverse audience levels and interests. 
+- Demonstrate your expertise in the topic with evidence of contributions to the Python community. Mention relevant publications, talks, or projects you would have worked on. 
+- Proofread your proposal for errors and clarity before submission.
+- Ensure that your talk content does not appear promotional. Keep it neutral and objective, discussing both commercial and open-source projects. Cover alternative perspectives and consider all sides of the topic. Use neutral language and avoid bias.
+- Adhere to the conference's code of conduct.
```
Comment (harisibrahimkv) [inline content/blog/2024/how-to-write-stellar-proposals.md:59]: Might be worthwhile linking to the CoC here?

Comment (Sc0rpi0n101) [reply content/blog/2024/how-to-write-stellar-proposals.md:59]:
```suggestion
- Adhere to the [conference's code of conduct](https://in.pycon.org/2024/code-of-conduct/).
```

**Review by anistark — APPROVED** [review]
Approved by @sayanchowdhury

**Pull request merged**